[2024-09-10T21:00:03.069-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T21:00:03.075-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl.weather_data scheduled__2024-09-10T00:00:00+00:00 [queued]>
[2024-09-10T21:00:03.079-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl.weather_data scheduled__2024-09-10T00:00:00+00:00 [queued]>
[2024-09-10T21:00:03.079-0300] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-10T21:00:03.092-0300] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): weather_data> on 2024-09-10 00:00:00+00:00
[2024-09-10T21:00:03.098-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_etl', 'weather_data', 'scheduled__2024-09-10T00:00:00+00:00', '--job-id', '483', '--raw', '--subdir', 'DAGS_FOLDER/daily_weather_dag.py', '--cfg-path', '/tmp/tmpf3hhwpxj']
[2024-09-10T21:00:03.099-0300] {standard_task_runner.py:105} INFO - Job 483: Subtask weather_data
[2024-09-10T21:00:03.100-0300] {logging_mixin.py:190} WARNING - /home/kasten/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=187108) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-10T21:00:03.100-0300] {standard_task_runner.py:72} INFO - Started process 187133 to run task
[2024-09-10T21:00:03.123-0300] {task_command.py:467} INFO - Running <TaskInstance: weather_etl.weather_data scheduled__2024-09-10T00:00:00+00:00 [running]> on host kasten-Aspire-A315-54
[2024-09-10T21:00:03.173-0300] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_etl' AIRFLOW_CTX_TASK_ID='weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-10T00:00:00+00:00'
[2024-09-10T21:00:03.173-0300] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T21:00:03.786-0300] {logging_mixin.py:190} INFO - Opened database successfully
[2024-09-10T21:00:03.799-0300] {logging_mixin.py:190} INFO - Close database successfully
[2024-09-10T21:00:03.799-0300] {python.py:240} INFO - Done. Returned value was: None
[2024-09-10T21:00:03.802-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T21:00:03.803-0300] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_etl, task_id=weather_data, run_id=scheduled__2024-09-10T00:00:00+00:00, execution_date=20240910T000000, start_date=20240911T000003, end_date=20240911T000003
[2024-09-10T21:00:03.839-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T21:00:03.846-0300] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-10T21:00:03.846-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
